### PR TITLE
Newsletters: add 202 (2022-06-01)

### PR DESCRIPTION
**Pull request by harding**
- [x] Add lede, update releases/RCs, and add topic entries @harding 
- [x] BTCPay Server 3772 @bitschmidty 
- [x] BTCPay Server 3744 @Xekyo 
- [x] BOLTs 968 @glozow 

(I'm sorry that all of the merge summaries this week are so trivial; I feel silly asking talented people to write obvious descriptions.)

### CODE REVIEW

**Comment by Xekyo** [comment]
No worries, we'll use the extra time we have this week to work on fancy research and spiffy pull requests ourselves.

File: _posts/en/newsletters/2022-06-01-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  either in csv or json format.
+
+- [BOLTs #968][] adds default TCP ports for nodes using Bitcoin testnet and
+  signet.
+
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="3772,3744,968,24897" %}
+[lnd 0.15.0-beta.rc3]: https://github.com/lightningnetwork/lnd/releases/tag/v0.15.0-beta.rc3
+[hwi 2.1.1]: https://github.com/bitcoin-core/HWI/releases/tag/2.1.1
+[news194 silent]: https://github.com/bitcoin-core/HWI/releases/tag/2.1.1
```
Comment (Xekyo) [inline _posts/en/newsletters/2022-06-01-newsletter.md:61]:
```suggestion
[news194 silent]: /en/newsletters/2022/04/06/#delinked-reusable-addresses
```

**Review by Xekyo — COMMENTED** [review]
Reviewed complete newsletter, tested links

File: _posts/en/newsletters/2022-06-01-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [BTCPay Server #3772][] allows users to turn on experimental features
+  for live-testing before release.
+
+- [BTCPay Server #3744][] adds a feature to export the wallet's transactions
+  either in csv or json format.
```
Comment (bitschmidty) [inline _posts/en/newsletters/2022-06-01-newsletter.md:52]:
```suggestion
  either in CSV or JSON format.
```

**Review by bitschmidty — COMMENTED** [review]
One additional minor suggestion, otherwise looks good!

File: _posts/en/newsletters/2022-06-01-newsletter.md (hunk 2 above)
Comment (Xekyo) [reply _posts/en/newsletters/2022-06-01-newsletter.md]: Fixed, thanks!

File: _posts/en/newsletters/2022-06-01-newsletter.md (hunk 1 above)
Comment (Xekyo) [reply _posts/en/newsletters/2022-06-01-newsletter.md]: I fixed this link in my commit just now as well.

**Comment by harding** [comment]
Added lede and topic links.  There have been no changes to the releases and RCs since this PR was opened.  Reviewed PR summaries---thanks @glozow @bitschmidty @Xekyo and thanks @Xekyo and @bitschmidty for the reviews!

**Comment by bitschmidty** [comment]
Thanks @harding @Xekyo and @glozow! 🚀

**Pull request merged**